include/fs/fs.h

```diff
@@ -45,30 +46,7 @@ enum fs_dir_entry_type {
 	FS_DIR_ENTRY_DIR
 };
 
-/** @brief Enumeration to uniquely identify file system types.
- *
- * Zephyr supports in-tree file systems and external ones.  Each
- * requires a unique identifier used to register the file system
- * implementation and to associate a mount point with the file system
- * type.  This anonymous enum defines global identifiers for the
- * in-tree file systems.
- *
- * External file systems should be registered using unique identifiers
- * starting at @c FS_TYPE_EXTERNAL_BASE.  It is the responsibility of
- * applications that use external file systems to ensure that these
- * identifiers are unique if multiple file system implementations are
- * used by the application.
- */
-enum {
-	/** Identifier for in-tree FatFS file system. */
-	FS_FATFS = 0,
-
-	/** Identifier for in-tree LittleFS file system. */
-	FS_LITTLEFS,
-
-	/** Base identifier for external file systems. */
-	FS_TYPE_EXTERNAL_BASE,
-};
+typedef int fs_type;
```

Comment:
Why was this change necessary?

Reply:
I wanted to use original FS enumeration but it would not work with DTS, I mean I could not do `mnttype = <FS_something>`